packages/cadl-python/src/emitter.ts

```diff
@@ -217,6 +217,7 @@ function getType(
     type: Type | CredentialType,
     modelTypeProperty: ModelProperty | undefined = undefined,
 ): any {
+    if (type.kind === "Model" && isAzureCoreErrorType(type)) return { type: "any" };
```

Comment:
This line is to avoid codegen failure in case the core Error are used as property of other models. (like the ADP cadl here: https://github.com/Azure/autorest.python/blob/dc280c0ef3c6119bc5caa50d1b07443baf195307/packages/cadl-python/adp/common/lro/common/models.cadl#L77)

Reply:
I talked to @johanste and the cadl is incorrect here, we don't have to generate this function at all (we'll just remove it form the client.cadl for now), so idt we need this code for now

Reply:
Hi @iscai-msft , updated and removed this line.